script/Arcueid/Day 6/06_00_ARC06_4.txt

```diff
@@ -182,6 +182,7 @@ An unexpected murder. An impossible reunion. An unbelievable monster.
 [sha:7ed0fcc21c54df4a379806f95d9ef00b78a77334]{
 -- Page 336, Offset 14747.
 -- 多くの死に背中を押されて伸ばした指と、死の手触り。
+// to be rechecked
```

Comment:
recheck here was surrounding "多くの死" and repetition of "death". Maybe
> Weighed down by that heavy loss of life, their outstretched fingers―――the touch of death.

Part of me wants to say "nasu needs a thesaurus!" but every time he _does_ use different words I go "i had no idea what this word means because my japanese sucks and thus this story is bad"